src/js/components/Card.js

```diff
@@ -150,10 +216,12 @@ export default class Card extends Component {
 
 Card.propTypes = {
   thumbnail: PropTypes.string,
-  description: PropTypes.string,
-  heading: PropTypes.string,
+  textSize: PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
   label: PropTypes.string,
-  link: PropTypes.element,
+  heading: PropTypes.string,
+  headingStrong: PropTypes.bool,
```

Comment:
What about supporting PropTypes.node as an option for 'heading' and having callers use `<strong>{...}</strong>` if they want a strong heading? Just thinking out loud.